src/lib.rs

```diff
@@ -17,6 +17,9 @@ use std::cmp::min;
 use std::result;
 use thiserror::Error;
 
+/// Default alignment that can be used for creating a `Constraint`.
+pub const DEFAULT_CONSTRAINT_ALIGN: u64 = 4;
```

Comment:
Do we also want to define a constructor that uses these defaults?

Reply:
I am not sure what do you mean by that. Do you mean to have a new constructor where we don't have to pass the default alignment?

Reply:
The default align and the default policy; I noticed `Constraint::new(0x1000, 0x1000, AllocPolicy::default())` and I was thinking if it makes sense to add a `Default` impl for Constraint to shorten things a bit there. Not sure if it makes sense, I don't have much context on vm-allocator.

Reply:
I find that to be useful mostly in tests (i.e. creating a constraint with default values). I created a variable for the align policy mostly as an example, but we can also delete it. The code looked a bit different in my first iteration and I needed that.